scripts/myscript.js

```diff
@@ -1,6 +1,6 @@
-function sayHello(fullname, id, language) {
-  console.log(
-    `Hello World, this is ${fullname} with HNGi7 ${id} using ${language}`
-  );
-}
-sayHello();
+function sayHello(fullname, id, language) {

```

Comment:
Look at this properly. You didn't pass the arguments you specified in the `sayHello` function

Reply:
that not mine